docs/midi_files.rst

```diff
@@ -181,46 +181,55 @@ The ``time`` attribute is used in several different ways:
 
 
-Tempo and Beat Resolution
+Tempo and Time Resolution
 -------------------------
 
 .. image:: images/midi_time.svg
 
-Timing in MIDI files is centered around ticks and beats. A beat is the same as 
-a quarter note. Beats are divided into ticks, the smallest unit of time in 
-MIDI.
+Timing in MIDI files is centered around ticks. Each message in a MIDI file has
+a delta time, which tells how many ticks have passed since the last message.
 
-Each message in a MIDI file has a delta time, which tells how many ticks have 
-passed since the last message. The length of a tick is defined in ticks per 
-beat. This value is stored as ``ticks_per_beat`` in MidiFile objects and 
-remains fixed throughout the song.
+A tick is the smallest unit of time in MIDI and remains fixed throughout the
+song. Each quarter notes is divided into a certain number of ticks, often
+referred as the resolution of the file or pulses per quarter note (PPQN). This
+resolution is stored as ``ticks_per_beat`` in MidiFile objects.
 
+The meaning of this ``ticks_per_beat`` in terms of absolute timing depends on
+the tempo and time signature of the file.
 
 MIDI Tempo vs. BPM
 ^^^^^^^^^^^^^^^^^^
 
-Unlike music, tempo in MIDI is not given as beats per minute, but rather in 
-microseconds per beat.
+Unlike music, tempo in MIDI is not given as beats per minute (BPM), but rather
+in microseconds per quarter note, with a default tempo of 500000 microseconds
+per quarter note. Given a default 4/4 time signature where a beat is exactly a
+quarter note, this corresponds to 120 beats per minute.
 
-The default tempo is 500000 microseconds per beat, which is 120 beats per 
-minute. The meta message 'set_tempo' can be used to change tempo during a song.
+In case of different time signatures, the length of a beat depends on the
+denominator of the time signature. E.g. in 2/2 time signature a beat has a
+length of a half note, i.e. two quarter notes. Thus the default MIDI tempo of
+500000 corresponds to a beat length of 1 second which is 60 BPM.
 
-You can use :py:func:`bpm2tempo` and :py:func:`tempo2bpm` to convert to and 
-from beats per minute. Note that :py:func:`tempo2bpm` may return a floating 
+The meta messages 'set_tempo' and 'time_signature' can be used to change
+the tempo and time signature during a song, respectively.
+
+You can use :py:func:`bpm2tempo` and :py:func:`tempo2bpm` to convert to and
+from beats per minute. Note that :py:func:`tempo2bpm` may return a floating
 point number.
 
 Converting Between Ticks and Seconds
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To convert from MIDI time to absolute time in seconds, the number of beats per 
-minute (BPM) and ticks per beat (often called pulses per quarter note or PPQ, 
-for short) have to be decided upon.
+To convert from MIDI time to absolute time in seconds, the tempo (either
+in number of beats per minute (BPM) or microseconds per quarter note, see
+above) and ticks per per quarter note have to be decided upon.
```

Comment:
`see above` should be a reST hypertext reference to the relevant section instead.